packages/workflow/src/records/search.ts

```diff
@@ -8,11 +8,19 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-import { Bundle, SavedTask, ValidRecord } from '@opencrvs/commons/types'
+import {
+  Bundle,
+  Resource,
+  SavedTask,
+  ValidRecord
+} from '@opencrvs/commons/types'
 import { SEARCH_URL } from '@workflow/constants'
 import fetch from 'node-fetch'
 
-export async function indexBundle(bundle: ValidRecord, authToken: string) {
+export async function indexBundle(
+  bundle: ValidRecord | Bundle<Resource>,
```

Comment:
Could we keep note of the places that will need to be changed once search stops accepting task bundles?